addons/mail/static/src/components/attachment_viewer/attachment_viewer.js

```diff
@@ -30,17 +31,13 @@ class AttachmentViewer extends Component {
                 attachmentViewer: attachmentViewer ? attachmentViewer.__state : undefined,
             };
         });
+        this._getRefs = useRefs();
```

Comment:
Add comment somewhere (probably here) why useRefs is needed instead of single ref (issue about t-key and t-ref)
And then add also comment why we need t-key in the first place (to force a new element because changing src doesn't work)